csvcubed/csvcubed/schema/codelist-config/v1_0/schema.json

```diff
@@ -0,0 +1,169 @@
+{
+  "$schema": "https://json-schema.org/draft-07/schema",
+  "id": "https://purl.org/csv-cubed/code-list-config/v1.0",
+  "title": "Schema for a codelist.",
+  "description": "Schema to be used for creation and validation of a single codelist in CSV-W format.",
+  "type": "object",
+  "required": [],
+  "properties": {
+    "$schema": {
+      "type": "string",
+      "default": "https://purl.org/csv-cubed/code-list-config/v1.0"
```

Comment:
I'd argue this is a `const` rather than a `default` but it hardly makes a difference.

Reply:
@robons , I agree too according to https://json-schema.org/understanding-json-schema/reference/generic.html. Default is more appropriate when/if there is more than one value in the field.

I have also updated the cube config schemas.